### DISCUSSION TITLE
Test generated wheels in a different container instance

### DISCUSSION THREAD

**Discussion by janaknat** [pypa/cibuildwheel#1360] [Q&A]
We are looking to build and test aarch64 wheels using GitHub CI and cibuildwheel (https://github.com/kivy/pyjnius/pull/639). As part of that, there is a requirement to build and test the wheels in separate containers. This is to 'prevent build script/installation from influencing tests (virtualenv is not enough sometimes) - https://github.com/kivy/pyjnius/pull/639#discussion_r1027311535. Is there an option that allows for the wheels to be only tested in a separate step? Currently, a build always takes place on using cibuildwheel.

**Comment by Czaki** [comment]
It is not possible currently. But You could provide more details in proper issue #317.
